testproject/Assets/Tests/Manual/HybridScripts/RpcQueueManualTests.cs

```diff
@@ -87,9 +106,79 @@ private enum NetworkManagerMode
         private ClientRpcParams m_ClientRpcParams;
         private ClientRpcParams m_ClientRpcParamsMultiParameter;
 
+        private ServerRpcParams m_MultiParameterTargets;
+
+
+        public void DisableManualNetworkManager()
+        {
+            if (m_ManualTestNetworkManager)
+            {
+                m_ManualTestNetworkManager.gameObject.SetActive(false);
+            }
+        }
+
+        public bool IsFinishedWithTest()
+        {
+            if (m_RunInTestMode)
+            {
+                return !m_ContinueToRun;
+            }
+            return false;
```

Comment:
if not running in test mode, shouldn't we finish immediately here and `return true`?

Reply:
public bool IsFinishedWithTest()
        {
            if (m_RunInTestMode)
            {
                return !m_ContinueToRun;
            }
            return false;
        }
        
Not really.  If you aren't running in test mode and you ask if you are finished with a test then that should return false.
As in: "I am not done with any test, because I am not in test mode".

Reply:
It is a test... :)